lua/nvim-tree/renderer/builder.lua

```diff
@@ -266,7 +266,15 @@ function Builder:create_combined_group(groups)
 
     -- build the highlight, overriding values
     for _, group in ipairs(groups) do
-      local hl = vim.api.nvim_get_hl(0, { name = group, link = false })
+      local hl
+      if vim.fn.has "nvim-0.8" == 1 then
+        hl = vim.api.nvim__get_hl_defs(0)[group] or {}
+        if hl["link"] then
```

Comment:
This check feels a little overkill considering we default to a table. I'm too much of a lua noob to know for sure.

Reply:
That's fine; safety first.